cmd/collector/app/builder/builder_flags.go

```diff
@@ -30,11 +30,15 @@ const (
 	collectorPort                 = "collector.port"
 	collectorHTTPPort             = "collector.http-port"
 	collectorGRPCPort             = "collector.grpc-port"
+	collectorTChanAddr            = "collector.tchan-server.host-port"
+	collectorHTTPAddr             = "collector.http-server.host-port"
+	collectorGRPCAddr             = "collector.grpc-server.host-port"
 	collectorGRPCTLS              = "collector.grpc.tls"
 	collectorGRPCCert             = "collector.grpc.tls.cert"
 	collectorGRPCKey              = "collector.grpc.tls.key"
 	collectorGRPCClientCA         = "collector.grpc.tls.client.ca"
-	collectorZipkinHTTPort        = "collector.zipkin.http-port"
+	collectorZipkinHTTPPort       = "collector.zipkin.http-port"
+	collectorZipkinHTTPAddr       = "collector.zipkin.host-port"
```

Comment:
can we also keep the var names consistent as collectorZipkinHTTPAddr-> collectorZipkinHTTPHostPort? It makes the code easier to read.